src/Workspaces/Core/Portable/Workspace/Solution/Document.cs

```diff
@@ -61,11 +61,6 @@ internal override bool HasInfoChanged(TextDocument otherTextDocument)
                    DocumentState.SourceCodeKind != otherDocument.SourceCodeKind;
         }
 
-        internal bool HasTextChanged(Document otherDocument)
```

Comment:
Is this an IVT break?

Reply:
Possibly, although I don't see any references to them on index. Just to be safe, I added them back as `obsolete`

Reply:
If you didn't find any then I'm OK with the delete; I just mostly wanted to make sure you checked.

Reply:
I will leave them as obsolete just in case. Will remove them later.